BJS2-41863-Уведомление о запросе рекомендации.

nue-01 opened the pull request:
Написал класс publisher вместе с юнит и инт. тестами; сделал исправления в сервисе запроса рекомендаций;

nue-01 committed:
BJS2-41863-Уведомление о запросе рекомендации.
Написал класс publisher вместе с юнит и инт. тестами; сделал исправления в сервисе запроса рекомендаций;

MiraGlaceon commented on src/main/resources/application.yaml:
```diff
@@ -30,6 +30,8 @@ spring:
         mentorship-requested-channel:
           name: mentorship_requested_channel
+        recommendation-requested-channel:
+          name: recommendation_requested_channel
```
Убери слово name, пожалуйста, везде. Один раз написали и теперь все повторяют. Name нужно либо вообще убрать, либо написать всего один раз уровнем выше

nue-01 committed:
BJS2-41863-Уведомление о запросе рекомендации.
Убрал в app.yaml свойство name у каждого канала, исправил ссылки на эти свойства у паблишеров.

nue-01 committed:
Merge branch 'Medusa-master-stream7' into feature/BJS2-41863

# Conflicts:
#	build.gradle.kts
#	src/main/java/school/faang/user_service/config/redis/RedisConfig.java
#	src/main/java/school/faang/user_service/service/recommendation/RecommendationRequestService.java
#	src/main/resources/application.yaml
#	src/test/java/school/faang/user_service/service/recommendation/RecommendationRequestServiceTest.java

nue-01 committed:
BJS2-41863-Уведомление о запросе рекомендации.
Решил мердж конфликты.

MiraGlaceon commented on src/main/resources/application.yaml:
```diff
@@ -21,22 +21,14 @@ spring:
-          name: ban_user
-        premium-bought-channel:
-          name: premium_bought_channel
+        mentorship-channel: mentorship_channel
```
Спасибо, как приятно стало созерцать
